docs/dc-v4-upgrade-guide.md

```diff
@@ -70,3 +70,11 @@ chart.on('renderlet', function (_chart) {
     - `dc.coordinateGridChart` --> `dc.coordinateGridMixin`
     - `dc.marginable` --> `dc.marginMixin`
     - `dc.stackableChart` --> `dc.stackMixin`
+
+- `dc.override` has been removed.
+   It was used to override a method in an object (typically a chart).
+   You will now need to extend the appropriate chart class.
```

Comment:
You can either create a derived class extending the chart class, or you can override specific methods with e.g.

```js
            chart.yAxisMin = () => {
                const min = d3.min(chart.data(), layer => d3.min(layer.values, p => p.y + p.y0));
                return dc.utils.subtract(min, chart.yAxisPadding());
            };
```

since methods on the object also override methods on the prototype chain.